pkg/document/crdt/tree.go

```diff
@@ -876,6 +876,34 @@ func (t *Tree) Style(from, to *TreePos, attributes map[string]string, editedAt *
 	return nil
 }
 
+// RemoveStyle removes the given attributes of the given range.
+func (t *Tree) RemoveStyle(from, to *TreePos, attributesToRemove []string, editedAt *time.Ticket) error {
+	// 01. split text nodes at the given range if needed.
+	fromParent, fromLeft, err := t.FindTreeNodesWithSplitText(from, editedAt)
+	if err != nil {
+		return err
+	}
+	toParent, toLeft, err := t.FindTreeNodesWithSplitText(to, editedAt)
+	if err != nil {
+		return err
+	}
+
+	err = t.traverseInPosRange(fromParent, fromLeft, toParent, toLeft,
+		func(token index.TreeToken[*TreeNode], _ bool) {
+			node := token.Node
+			if !node.IsRemoved() && !node.IsText() && len(attributesToRemove) > 0 && node.Attrs != nil {
+				for _, value := range attributesToRemove {
+					node.Attrs.Remove(value, editedAt)
```

Comment:
`RHT.Remove` has never been used and seems buggy.

The problem I found is that when the code below is called, a panic occurs because `Node.removedAt` is nil.

https://github.com/yorkie-team/yorkie/blob/3b712df6c3d03029a1cf565070f0cfc2bf960abe/pkg/document/crdt/rht.go#L116-L124

I think we need to add tests for concurrent value setting and deletion in RHT.
https://github.com/yorkie-team/yorkie/blob/main/pkg/document/crdt/rht_test.go

Reply:
I fixed it. I found that after remove element, return value of `RHT.Len` is incorrect, and I have also fixed this issue.